library/build.gradle

```diff
@@ -34,6 +35,9 @@ android {
         installOptions "-d", "-t"
     }
 
+    defaultConfig {
+        testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
+    }
```

Comment:
Tests were not found without the runner. No idea how it worked before (if it did).